docs/usage/kafka_containers.md

```diff
@@ -0,0 +1,51 @@
+# Kafka Containers
+
+Testcontainers can be used to automatically instantiate and manage [Apache Kafka](https://kafka.apache.org) containers.
+More precisely Testcontainers uses official Docker images for [Confluent OSS Platform](https://hub.docker.com/r/confluentinc/cp-kafka/)
+
+## Benefits
+
+* Running single node Kafka installation with just one line of code
+* No need to manage external Zookeeper installation, required by Kafka. But see [below](#zookeeper)
+
+## Example
+
+The following field in your JUnit test class will prepare a container running Kafka:
+```java
+@Rule
+public KafkaContainer kafka = new KafkaContainer();
+```
+        
+Now your tests or any other process running on your machine can get access to running Kafka broker by using the following bootstrap server location:
+```java
+kafka.getBootstrapServers()
+```
+
+## Options
+
+### Selecting Kafka version
+
+You can select a version of Confluent Platform by passing it to container's constructor:
+```java
+new KafkaContainer("4.1.2")
+```
+The correspondence between Confluent Platform versions and Kafka versions can be seen [in Confluent documentation](https://docs.confluent.io/current/installation/versions-interoperability.html#cp-and-apache-kafka-compatibility)
+        
+### <a name="zookeeper"></a> Using external Zookeeper
+
+If for some reason you want to use an externally running Zookeeper, then just pass its location during construction:
```

Comment:
just provide it's location

Reply:
I kindly disagree on this one :) https://data.grammarbook.com/blog/pronouns/1-grammar-error/

Reply:
You are 100% correct of course :D